qnap/shared/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "log_level": "trace",
+    "log_file_path": "/tmp/nordsecuritymeshnet/teliod.log",
+    "authentication_token": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+    "app_user_uid": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
+    "interface_name": "utun0"
```

Comment:
it would porably be better if it's nlx or nordlynx

Reply:
will change